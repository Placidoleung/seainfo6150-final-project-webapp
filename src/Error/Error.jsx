import React from 'react'
import NavBar from "../NavBar/NavBar.jsx";
import styles from './Error.module.css';


const Error = () => {
    return (
        <div className={styles.base}>
            <NavBar></NavBar>
            <div className={styles.container}>
                <h1 className={styles.warn}>
                    Oops...That page couldn't be found.
                </h1>
                <a className={styles.back} href="/">Go back to home page?</a>
            </div>
        </div>
    )
}

export default Error
