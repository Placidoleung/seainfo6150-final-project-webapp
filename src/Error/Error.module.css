@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Orelega+One&display=swap');

.container {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    background-color: #333;
    height: 800px;
}

.warn {
    color: white;
    font-size: 80px;
    font-family: 'Orelega One', cursive;
    font-weight: normal;
}

.back{
    color: white;
    font-size: 40px;
    font-family: 'Montserrat', sans-serif;
}