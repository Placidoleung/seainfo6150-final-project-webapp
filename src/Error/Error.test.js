import React from 'react'
import Error from './Error'

describe("Error tests", () => {

    it('render correctly ', () => {
        const { container } = render(<Error />);
        expect(container).toMatchSnapshot();
    });

});