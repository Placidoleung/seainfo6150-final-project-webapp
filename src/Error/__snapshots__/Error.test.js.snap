// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error tests render correctly  1`] = `
<div>
  <div
    class="base"
  >
    <div
      class="container"
    >
      <div
        class="logo"
      >
        <a
          href="/"
        >
          BlockNews
        </a>
      </div>
      <ul
        class="nav"
      >
        <li
          class="item"
        >
          <a
            href="/"
          >
            Home
          </a>
        </li>
        <li
          class="item"
        >
          <a
            href="/contact"
          >
            Contact
          </a>
        </li>
        <li
          class="item"
        >
          <a
            href="/about"
          >
            About
          </a>
        </li>
      </ul>
    </div>
    <div
      class="container"
    >
      <h1
        class="warn"
      >
        Oops...That page couldn't be found.
      </h1>
      <a
        class="back"
        href="/"
      >
        Go back to home page?
      </a>
    </div>
  </div>
</div>
`;
